default.nix

```
@@ -56,4 +56,17 @@ stdenv.mkDerivation {
     ++ optionals ecDeps [ easycrypt alt-ergo z3.out ]
     ++ optionals opamDeps [ rsync git pkg-config perl ppl mpfr opam ]
     ;
+
+  buildPhase = ''
+    make -C compiler CIL
+    make -C compiler
+  '';
+
```

```suggestion
  enableParallelBuilding = true;
```

The default `buildPhase` is good enough.